contracts/coordinator/package.json

```diff
@@ -71,7 +71,7 @@
         "@0x/base-contract": "^5.1.0",
         "@0x/contracts-asset-proxy": "^2.1.5",
         "@0x/contracts-erc20": "^2.2.5",
-        "@0x/contracts-exchange": "1.0.2",
+        "@0x/contracts-exchange": "^2.1.5",
```

Comment:
Every time we run `contracts:build` from monorepo root, all the generated wrappers in all of the contracts packages are re-generated and pick up the changes made in `abi-gen`/`abi-gen-templates`/`abi-gen-wrappers`. Even though these generated wrappers aren't committed, they are exported and other contracts packages depend on them.

`contracts-coordinator` was pinned to an old version of `contracts-exchange` which was cached in `node_modules` and not picking up any of the `abi-gen` changes. The pinning seemed unintentional because `contracts-exchange` is only imported in test files in `contracts-coordinator`, so I updated it.